src/updi_nvm.c

```diff
@@ -1065,6 +1065,7 @@ int updi_nvm_chip_erase(const PROGRAMMER *pgm, const AVRPART *p) {
     case UPDI_NVM_MODE_V2:
       return nvm_chip_erase_V2(pgm, p);
     case UPDI_NVM_MODE_V3:
+    case UPDI_NVM_MODE_V5:
```

Comment:
...this one, and all the following changes, are not needed

Reply:
this will revert